playbooks/commontemplatesbundle.yaml

```diff
@@ -1,5 +1,9 @@
 - hosts: localhost
   gather_facts: no
   tasks:
-  - import_role:
+  - name: Gather informations about the repository
```

Comment:
I think you can also use just:
roles:
- KubevirtRepoInfo
- KubevirtCommonTemplatesBundle

See https://docs.ansible.com/ansible/latest/user_guide/playbooks_reuse_roles.html#role-duplication-and-execution

Reply:
Thanks, will fix.